de-buck.csl

```diff
@@ -20,18 +20,21 @@
     <category citation-format="note"/>
     <category field="generic-base"/>
     <summary>Annotatie en literatuurlijst volgens P. de Buck e.a., Zoeken en schrijven. Handleiding bij het maken van een historisch werkstuk (Haarlem 1982). Gebaseerd op "MHRA format with full notes and bibliography"</summary>
-    <updated>2013-01-20T12:26:52+01:00</updated>
+    <updated>2016-05-04T17:23:52+01:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="nl">
     <terms>
       <term name="et-al">e.a.</term>
-      <term name="editor" form="short">
+      <term name="editor" form="verb-short">
         <single>ed.</single>
-        <multiple>eds.</multiple>
+        <multiple>ed.</multiple>
       </term>
       <term name="edition" form="short">druk</term>
-      <term name="translator" form="verb-short">(vert.)</term>
+      <term name="translator" form="verb-short">
+        <single>(vertaler)</single>
+        <multiple>(vertalers)</multiple>
```

Comment:
If you don't mind, it's better to move the parenthesis out of the term defitions, and into the label (`<label ... prefix="(" suffix=")"/>`).